packages/composites/multi-address-list/src/endpoint/address-list.ts

```diff
@@ -0,0 +1,90 @@
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { config } from '../config'
+import { addressListTransport } from '../transport/address-list'
+import {
+  PoRAddressEndpoint,
+  PoRAddressResponse,
+} from '@chainlink/external-adapter-framework/adapter/por'
+import { walletParameters as anchorageParams } from '@chainlink/anchorage-adapter'
+import { walletParameters as coinbasePrimeParams } from '@chainlink/coinbase-prime-adapter'
+import { walletParameters as bitGoParams } from '@chainlink/bitgo-adapter'
+
+export const inputParameters = new InputParameters(
```

Comment:
I felt there is just too much hard-coding here, adding another EA here would require us to make changes but we are essentially just doing a pass-through here